newdle/api.py

```diff
@@ -1,12 +1,13 @@
 import uuid
 from importlib import import_module
 
+from pytz import common_timezones_set
```

Comment:
Maybe the environment was not activated and this was automatically sorted like this? (I recall I had the same problem or similar)

Reply:
I think using `isort` from outside the env could cause the same issue. And I just realized again that we aren't running isort during CI (see #172). Since that's a small one maybe it'd be nice to fix that issue first and then rebase this PR once it's merged to have the CI checks run on the changes here?

Reply:
It was most likely due to the environment not being activated. As running it again restored the original formatting.